ts/components/portfolio/CreditCardComponent.tsx

```diff
@@ -0,0 +1,141 @@
+
+import I18n from "../../i18n"
+import ROUTES from "../../navigation/routes"
+
+import { CreditCard } from "../../types/portfolio/CreditCard"
+import { CreditCardType } from "../../types/portfolio/CreditCardType"
+import { CreditCardStyle } from '../styles'
+
+type Props = {
+  item: CreditCard,
+  navigation: NavigationScreenProp<NavigationState>
+}
+
+// Images
+const imgMastercard = require("../../../img/portfolio/issuers/mastercard.png")
+const imgVisa = require("../../../img/portfolio/cards-icons/visa.png")
+const imgAmex = require("../../../img/portfolio/cards-icons/amex.png")
+const imgDiners = require("../../../img/portfolio/cards-icons/amex.png")
+const imgMaestro = require("../../../img/portfolio/cards-icons/amex.png")
+const imgElectron = require("../../../img/portfolio/cards-icons/visa-electron.png")
+
+/**
+ * Credit card component
+ */
+export default class CreditCardComponent extends React.Component<Props> {
+
+  public render(): React.ReactNode {
+    const { item } = this.props;
+    const { navigate } = this.props.navigation;
+    return (
+      <Card
+      style={ CreditCardStyle.cardStyle } >
+      <Body>
+        <Grid>
+          <Row size={2} style = {CreditCardStyle.rowStyle}>
+            <Col size={7}>
+              <Text style = { [CreditCardStyle.textStyle, CreditCardStyle.largeTextStyle] } >
+                {("\u25cf".repeat(4) + " ").repeat(3) + item.number.slice(-4)}
+              </Text>
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={false}
+                style={CreditCardStyle.iconStyle}
+                name="star-border"
+              />
+            </Col>
+            <Col size={1}>
+              <Icon
+                type = "MaterialIcons"
+                active={true}
+                style={CreditCardStyle.iconStyle}
+                name="more-vert"
+              />
+            </Col>
+          </Row>
+          <Row size={1} style = {CreditCardStyle.rowStyle}>
+            <Col>
+              <Text style = { [CreditCardStyle.textStyle, CreditCardStyle.smallTextStyle] } >
+                {I18n.t('creditCardComponent.validUntil') + item.expires}
+              </Text>
+            </Col>
+          </Row>
+          <Row size={2} style = {[CreditCardStyle.rowStyle, CreditCardStyle.whiteBarStyle]}>
+            <Col size={7}>
+              <Text style = { CreditCardStyle.textStyle }>
+                {item.owner.toUpperCase()}
+              </Text>
+            </Col>
+            <Col size={2}>
+              <Image
+                style = {CreditCardStyle.issuerLogo}
+                source= { this.getCreditCardImage(item.number) }/>
+            </Col>
+            <Col size={1}><Text>&nbsp;</Text></Col>
+          </Row>
+          {/* onPress supported by row, but not recognized by typescript
+            * (https://github.com/GeekyAnts/react-native-easy-grid/blob/master/Components/Row.js#L29)
```

Comment:
should we add the `onPress` prop to the `Row` type? - cc @fpersico

Reply:
Yes because RowProps interface now only extends ViewProperties that doesn't provide onPress prop.